lib/driver.js

```diff
@@ -695,6 +695,10 @@ class XCUITestDriver extends BaseDriver {
       this._perfRecorders = [];
     }
 
+    if (this._conditionInducerService) {
```

Comment:
Please also add `this._conditionInducerService = null` into the driver constructor